components/brave_ads/browser/ads_service_impl.cc

```diff
@@ -904,7 +901,14 @@ void AdsServiceImpl::SetIdleThreshold(const int threshold) {
 
 bool AdsServiceImpl::IsNotificationsAvailable() const {
 #if BUILDFLAG(ENABLE_NATIVE_NOTIFICATIONS)
+
+#if defined(OS_ANDROID)
+   const NotificationHelper * notification_helper = NotificationHelper::GetInstance();
+   return notification_helper->IsNotificationsAvailable() && notification_helper->IsBraveAdsChannelEnabled();
```

Comment:
At this time only IsNotificationsAvailable should be part of the helper, as the helper classes are for cross platform implementations and IsBraveAdsChannelEnabled is specific to Android

Reply:
@tmancey 
1. `IsBraveAdsChannelEnabled` is now a private member of `NotificationHelperAndroid`.
2.  Java `OpenPageFromNative` is substituted by native `ServiceTabLauncher::LaunchTab`.